deployment/artifacts/mainnet.json

```diff
@@ -1499,7 +1499,7 @@
             "tx_hash": "0x58617564fa2bf9ad71b9eab1ceefdd3eae5a46efc71f4463df6dbffbd713a160",
             "block_number": 18622368,
             "deployer": "0xFfFd7092685bDeeBD121D1A0FEA3c349114Cce50"
-        },
+        }
```

Comment:
Thanks @piotr-roslaniec - my bad.

Reply:
Was this merged manually or with the cli?

Reply:
The original merge was done with the tool (new contracts and legacy contracts), but then I needed to remove the legacy contracts after you and @vzotova weighed in so I did that manually. 

Of course, I can redo the merge entirely to exclude the deprecated contracts if that would be more reassuring...

Reply:
Let me redo it locally and see if the files match...